docs/apis-clients/working-with-apis-clients.md

```diff
@@ -60,6 +60,33 @@ Community clients supplement the official clients. These clients have not been t
 - [Spring](community-clients/spring.md)
 - [Quarkus](community-clients/quarkus.md)
 
-You can browse other community extensions and the most up-to-date list of community clients [here](https://github.com/orgs/camunda-community-hub/repositories).
+It is also possible to [build your own client](build-your-own-client.md) You can browse other community extensions and the most up-to-date list of community clients [here](https://github.com/orgs/camunda-community-hub/repositories).
+
+## Learn about Camunda Components and their APIs
+
+![Architecture diagram for Camunda Platform including all the components for SaaS](./img/ComponentsAndArchitecture_SaaS.png)
+
+<DocCardList items={[{type:"link", href:"/docs/apis-clients/tasklist-api/tasklist-api-overview/", label: "Tasklist API (GraphQL)", docId:"apis-clients/tasklist-api/tasklist-api-overview"},
+{
+type:"link", href:"/docs/apis-clients/public-api/", label: "Public API", docId:"apis-clients/public-api"
+},
+{
+type:"link", href:"/docs/apis-clients/operate-api/index/", label: "Operate API (REST)", docId:"apis-clients/operate-api/index"
+},
+{
+type:"link", href:"/docs/apis-clients/console-api-reference/", label: "Console API clients (REST)", docId:"apis-clients/console-api-reference"
```

Comment:
Why does it state "Console API **clients** (REST)"?

Reply:
Nice catch. Adjusted in commit 👍